test/integration/Cython_0001-Pyston-change-we-don-t-support-custom-traceback-entr.patch

```diff
@@ -121,5 +138,30 @@ index 0310570..bcd0eb2 100644
      }
  #if CYTHON_COMPILING_IN_CPYTHON
      assert(PyType_IS_GC(self->ob_type) &&
+diff --git a/Cython/Utility/ModuleSetupCode.c b/Cython/Utility/ModuleSetupCode.c
+index 6477fb2..a30334c 100644
+--- a/Cython/Utility/ModuleSetupCode.c
++++ b/Cython/Utility/ModuleSetupCode.c
+@@ -49,13 +49,16 @@
+ 
+ #if PY_MAJOR_VERSION < 3
+   #define __Pyx_BUILTIN_MODULE_NAME "__builtin__"
+-  #define __Pyx_PyCode_New(a, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos) \
+-          PyCode_New(a+k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
++  // Pyston change: we don't support this yet
++  // #define __Pyx_PyCode_New(a, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos) \
++  //         PyCode_New(a+k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
++  #define __Pyx_PyCode_New(a, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos) None
```

Comment:
This feels risky to have this silently just return None -- what was using it?  Could we modify the caller(s) instead?